### PR TITLE
feat(iotda/device_proxies): support device proxies datasource

### PR DESCRIPTION
**Pull request by jinyangyang222**
**What this PR does / why we need it**:

Support device proxies datasource

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.

```
$ export HW_IOTDA_ACCESS_ADDRESS=xxxxxxxxxxxxx

$ make testacc TEST="./huaweicloud/services/acceptance/iotda" TESTARGS="-run TestAccDataSourceDeviceProxies_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/iotda -v -run TestAccDataSourceDeviceProxies_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceDeviceProxies_basic
=== PAUSE TestAccDataSourceDeviceProxies_basic
=== CONT  TestAccDataSourceDeviceProxies_basic
--- PASS: TestAccDataSourceDeviceProxies_basic (14.64s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/iotda     14.693s

```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: docs/data-sources/iotda_device_proxies.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+subcategory: "IoT Device Access (IoTDA)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_iotda_batchtasks"
```
Comment (deer-hang) [inline docs/data-sources/iotda_device_proxies.md]: Please enter the correct `page_title`.

Comment (jinyangyang222) [reply docs/data-sources/iotda_device_proxies.md]: fixed.

**Pull request merged**